### PR TITLE
Fetching config using k8s client

### PR DESCRIPTION
**Pull request by GunaKKIBM**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -49,57 +51,20 @@ type Config struct {
...
-	if confPath := getEnv("SECRET_CONFIG_PATH"); confPath != "" {
-		return filepath.Join(confPath, "libconfig.toml")
-	}
-	//Get default conf path
-	return GetDefaultConfPath()
-}
-
-// GetConfPathDir get configuration  dir path
-func GetConfPathDir() string {
-	if confPath := getEnv("SECRET_CONFIG_PATH"); confPath != "" {
```
Comment (ambiknai) [inline config/config.go:79]: Please make sure these methods are unused.

GetConfPathDir,  GetDefaultConfPath, GetConfPath etc.

**Review by sameshai — APPROVED** [review]

File: config/config.go (hunk 1 above)
Comment (GunaKKIBM) [reply config/config.go:79]: Only https://github.com/IBM/ibm-vpc-file-csi-driver/blob/d89edd7b27f251b84b17c002c1294bab0abd4f3f/cmd/main.go#L95 - here this is getting used, but will be updating file csi driver code also soon.

**Review by arahamad — APPROVED** [review]
please verify these changes on all supported envs

**Review by prankulmahajan — APPROVED** [review]
lgtm

**Pull request merged**